dev-requirements.txt

```diff
@@ -19,6 +20,7 @@ pytest-logbook
 pytest-csv
 pytest-xdist
 pytz
+snowflake-connector-python
```

Comment:
See other comment about `cryptography`. Same applies to `snowflake-connector-python`.